python/invoke_release/tasks.py

```diff
@@ -1503,3 +1517,28 @@ def wheel(_):
         archive_name=archive_name,
         base_dir=base_dir
     ))
+
+
+def open_pull_request(options, token, gh_owner,gh_repo):
+    url = 'https://api.github.com/repos/{gh_owner}/{gh_repo}/pulls'
+    values = {
+      'title': options.title,
+      'base': options.base,
+      'head': options.base.head,
+      'body': options.body
+    }
+
+    data = json.dumps(values).encode('utf-8')
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': 'token {}'.format(token),
+        'Accept': 'application/vnd.github.v3+json',
+        'Content-Length': len(data)
+    }
+
+    try:
+        req = urllib.request.Request(url, data, headers)
+        with closing(urllib.request.urlopen(req)) as f:
+            res = f.read()
+    except Exception as e:
+        print(e)
```

Comment:
We should use `_error_output('Could not open a Pull Request')` in here instead of just printing the error.